src/v/pandaproxy/client/client.cc

```diff
@@ -72,22 +72,24 @@ ss::future<> client::stop() {
 }
```

Comment:
Please describe in the commit message the scenario that prompted this fix. For example, is it a shutdown sequence? Are some requests dispatched in the background, causing the lifetime issue? If this is related to the fetch session assertion we saw earlier, that explanation will be doubly useful.

Reply:
@BenPope was this related to the fetch session cache bug?

Reply:
#178 or the failure on `test_raft_rpunit` ?